packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/components/TableComponent.java

```diff
@@ -56,6 +58,16 @@ public Element render() {
         TableRenderingCache cache = new TableRenderingCache();
         ITableElementRequestor tableElementRequestor = new TableElementRequestor();
 
+        AtomicReference<String> globalFilter = new AtomicReference<>("");
```

Comment:
You don't need a concurrent data structure for that

Reply:
It's funny because this line is making sure that we will never really have null as the value of the global filter... It does not have to be really nullable

Reply:
I need to handle null value as global search to differentiate the case where we create a new representation and we don't know yet the global filter value and the case where we remove the current global value (in this case '')